src/serlo-editor-repo/plugin-rows/editor/index.tsx

```diff
@@ -82,7 +88,19 @@ export function RowsEditor(props: RowsProps) {
 
   if (!props.editable) return <RowsRenderer {...props} />
 
-  const isEditorForRootOfDocument = props.id === 'root'
+  // Prevent add button being visually emphasized when this RowsEditor is contained within certain plugin types.
+  const visuallyEmphasizeLastAddButton =
```

Comment:
This means any time we introduce a plugin that contains a row plugin we need to think about changing this. 
I see you already tried, but just to be sure: Is there a feasible way identify the root without checking all plugin types?

Reply:
> This means any time we introduce a plugin that contains a row plugin we need to think about changing this.
I see you already tried, but just to be sure: Is there a feasible way identify the root without checking all plugin types?

I tried a few things but no simple solution like `props.is === 'root'` worked in all content types. For example, in type exercise we have two RowsEditors nested like this: 
- "type-text-exercise", "exercise", "rows"
- "type-text-exercise", "type-text-solution", "solution", "rows"

The fact that they are at nested at different levels and that other content types each have their own nesting structure made it difficult for me to find a solution that worked everywhere and does not rely on checking the plugin types on the path to the root.

Reply:
thanks for checking!